src/index.js

```diff
@@ -1,3 +1,5 @@
 // @flow
 export {default as withFontLoading} from './with-font-loading';
```

Comment:
@angus-c - thoughts on making this one named, and `FontPlugin` below the default?

Reply:
sounds good